<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="Firstnameline">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>301</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
	border: 2px solid #000000;
	border-radius: 5px;
	padding: 1px;
	box-shadow: 0 0 10px rgba(0,0,05);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Genderline">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
	border: 2px solid #000000;
	border-radius: 5px;
	padding: 1px;
	box-shadow: 0 0 10px rgba(0,0,05);</string>
    </property>
    <item>
     <property name="text">
      <string>Male</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Female</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="FirstName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Middlenameline">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>301</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
	border: 2px solid #000000;
	border-radius: 5px;
	padding: 1px;
	box-shadow: 0 0 10px rgba(0,0,05);</string>
    </property>
   </widget>
   <widget class="QLabel" name="MiddleName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Middle Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Lastnameline">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>301</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
	border: 2px solid #000000;
	border-radius: 5px;
	padding: 1px;
	box-shadow: 0 0 10px rgba(0,0,05);</string>
    </property>
   </widget>
   <widget class="QLabel" name="LastName">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="Gender">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Gender</string>
    </property>
   </widget>
   <widget class="QLabel" name="YearLevel">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>330</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Year Level</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Yearlevelline">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>350</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
	border: 2px solid #000000;
	border-radius: 5px;
	padding: 1px;
	box-shadow: 0 0 10px rgba(0,0,05);</string>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="IDnumber">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>280</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>I.D. Number</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IDline">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>300</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
	border: 2px solid #000000;
	border-radius: 5px;
	padding: 1px;
	box-shadow: 0 0 10px rgba(0,0,05);</string>
    </property>
    <property name="placeholderText">
     <string>e.g. 2000-0000</string>
    </property>
   </widget>
   <widget class="QLabel" name="StudentProfile">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
    border: 2px solid #800000;
    border-radius: 10px;
    padding: 1px;
    box-shadow: 0 0 10px rgba(0, 0, 0, 0.5);
    color: black;</string>
    </property>
    <property name="text">
     <string>Student Profile</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="CourseCode">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Rockwell</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Course Code</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Edit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>390</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #800000;
    color: white;
    border: 2px solid #000000;
    border-radius: 5px;
     padding: 1px;
    box-shadow: 0 0 10px rgba(0, 0, 0, 0.5);
    color: white; 
}

/ Hover effect for the button */
QPushButton:hover {
    background-color: #944E63;
    border: 2px solid #944E63;
}</string>
    </property>
    <property name="text">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>390</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>91</width>
      <height>20</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    background-color: #800000;
    color: white;
    border: 2px solid #000000;
    border-radius: 5px;
     padding: 1px;
    box-shadow: 0 0 10px rgba(0, 0, 0, 0.5);
    color: white; 
}

/ Hover effect for the button */
QPushButton:hover {
    background-color: #944E63;
    border: 2px solid #944E63;
}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>390</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover {
    background-color: #808080;
    border: 2px solid #2980b9;
}</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Save">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton:hover {
    background-color: #808080;
    border: 2px solid #2980b9;
}</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>651</width>
      <height>491</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLineEdit" name="SearchLine">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>301</width>
       <height>21</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: white;
	border: 2px solid #000000;
	border-radius: 5px;
	padding: 1px;
	box-shadow: 0 0 10px rgba(0,0,05);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Search">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>91</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    background-color: #800000;
    color: white;
    border: 2px solid #000000;
    border-radius: 5px;
     padding: 1px;
    box-shadow: 0 0 10px rgba(0, 0, 0, 0.5);
    color: white; 
}

/ Hover effect for the button */
QPushButton:hover {
    background-color: #944E63;
    border: 2px solid #944E63;
}</string>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="TabTable">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>611</width>
       <height>411</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Students</string>
      </attribute>
      <widget class="QTableView" name="StudentTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>611</width>
         <height>401</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Program</string>
      </attribute>
      <widget class="QTableView" name="ProgTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>601</width>
         <height>411</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>College</string>
      </attribute>
      <widget class="QTableView" name="CollegeTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>601</width>
         <height>381</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QComboBox" name="Sort">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>ID number</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lastname</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QComboBox" name="CourseComboBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>141</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;
	border: 2px solid #000000;
	border-radius: 5px;
	padding: 1px;
	box-shadow: 0 0 10px rgba(0,0,05);</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
